autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
```

Comment:
Approving, but remember to fix this linter warning before merging

Reply:
To be honest I thought the GitHub Actions frontend linting step would fail with warnings (so I assumed I was good lol), is that something we want to add?